### ISSUE TITLE
Pure nix shell has bazel as a command not found.

### ISSUE THREAD

**Issue by philderbeast** [tweag/rules_haskell#432]
Following the [for developers](https://github.com/tweag/rules_haskell#for-rules_haskell-developers) section of the readme, bazel is a command not found;

> To run the test suite for these rules, you'll need Nix installed. First, from the project’s folder start a pure nix shell:

```
> git clone https://github.com/tweag/rules_haskell.git
> cd rules_haskell
> bazel version
Starting local Bazel server and connecting to it...
Build label: 0.16.1
...
Build time: Mon Aug 13 13:42:50 2018 (1534167770)
Build timestamp: 1534167770
Build timestamp as int: 1534167770
> nix-shell --pure shell.nix
[nix-shell:~/.../rules_haskell]$ bazel test //...
bash: bazel: command not found
```

```
> nix --version
nix (Nix) 2.1.1
> sw_vers
ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G65
```

**Comment by Profpatsch** [comment]
https://github.com/tweag/rules_haskell/blob/master/shell.nix#L41-L43

Since bazel works on Darwin now, we can add it unconditionally.

**Comment by guibou** [comment]
Recently merged #453 now pulls `bazel` from `nixpkgs`, so `nix-shell --pure` should work as expected.
